Equinox/Plane.cpp

```diff
@@ -24,7 +24,7 @@ void ::Plane::Draw()
 	float3 axis = Rotation.Axis();
 	glRotatef(RadToDeg(Rotation.Angle()), axis.x, axis.y, axis.z);
 
-	glColor4f(0.f, 0.f, 0.f, 0.f);
+	glColor4f(Color.x, Color.y, Color.z, 0.f);
```

Comment:
Nope, it's a float3 var... we can change it to all the primitives (and in Primitive.h of corse) later to maintain the concistence of the project.